userguide/activity_classifier/export_coreml.md

```diff
@@ -1,15 +1,142 @@
 # Deploying to Core ML
 
-In this section, we describe how to export the model created by the `activity_classifier` toolkit to Core ML, and use it in Xcode to get real time predictions in a swift app.
+In this section, we describe how to export the model created by the activity classifier toolkit to Core ML, and use it in Xcode to get real-time predictions in a Swift app.
 
 Exporting an activity classifier model in Core ML format can be performed using the `export_coreml` function.
 
 ```python
 model.export_coreml('ActivityClassification.mlmodel')
 ```
-Dragging the saved model to Xcode will automatically generate relevant code for calling the model in your app:  
 
-![Model imported to Xcode](images/Model_In_Xcode.png)
+## Deployment With Turi Create 6
+
+Starting with Turi Create 6.0 the API for integrating an Activity Classifier into your Swift app has become more explicit. There are now individual features for the accelerometer and gyroscope on x, y, and z axes. Dragging the saved model to Xcode will automatically generate relevant code for calling the model in your app:
+
+![Model imported to Xcode](images/Model_In_Xcode_6.0.png)
+
+For more information about the generated programmatic API please refer to [Core ML documentation](https://developer.apple.com/documentation/coreml/integrating_a_core_ml_model_into_your_app).
+
+##### Model Inputs
+
+* **acc_x** - an array in the length of ```prediction_window``` containing the accelerometer's X-axis readings.
+* **acc_y** - an array in the length of ```prediction_window``` containing the accelerometer's Y-axis readings.
+* **acc_z** - an array in the length of ```prediction_window``` containing the accelerometer's Z-axis readings.
+* **gyro_x** - an array in the length of ```prediction_window``` containing the gyroscope's X-axis readings.
+* **gyro_y** - an array in the length of ```prediction_window``` containing the gyroscope's Y-axis readings.
+* **gyro_z** - an array in the length of ```prediction_window``` containing the gyroscope's Z-axis readings.
+* **stateIn** - cell state input for the model's LSTM recurrent layer. Initialize to zero when starting a new session, otherwise should be fed with the last `stateOut` output of the previous prediction.
+
+##### Model Outputs
+
+* **activityProbability** - a dictionary, where the keys are each possible label (predicted activity) and the values are the activity's predicted probability - a double in the range [0.0, 1.0].
```

Comment:
Do we want to replace the second dash in this sentence with either a double dash or a colon?